test/read_only_api_tests.jl

```diff
@@ -41,6 +47,13 @@ auth = authenticate(string(circshift(["bcc", "3fc", "03a", "33e",
 # test GitHub.repos
 @test hasghobj(ghjl, first(repos(julweb; auth = auth)))
 
+# test pubkey retrieval
+@test first(values(GitHub.pubkeys(testuser)[1])) == testuser_pubkey
```

Comment:
`GitHub.pubkeys(testuser; auth = auth)`, so that testing doesn't hit GitHub's unauthenticated rate limit.

This change also needs to be made for the `check_membership` test. Thanks!